notebooks/lfqa/retriever_reader_for_lfqa_predict_mode.ipynb

```diff
@@ -199,8 +199,8 @@
     }
    ],
    "source": [
-    "from primeqa.components.retriever.dense import ColBERTRetriever\n",
-    "from primeqa.components.reader.generative import GenerativeFiDReader\n",
+    "from primeqa.Components.retriever.dense import ColBERTRetriever\n",
```

Comment:
Is the capitalization in "Components" right?

Reply:
changed it to lowercase.